nix_update/version/gitlab.py

```diff
@@ -29,7 +31,9 @@ def fetch_gitlab_version(url: ParseResult, version_regex: str) -> Optional[str]:
             name = tag["name"]
             assert isinstance(name, str)
             extracted = extract_version(name, version_regex)
-            if extracted is not None:
+            if extracted is not None and (
+                (not version_stable) or version_is_stable(extracted)
```

Comment:
I remember that the github also had a field if the version is stable.

Reply:
In the API, yes. Not in the Atom feed.